### PR TITLE
Updated cluster status for 4.17 upi 

### PR DESCRIPTION
**Pull request by alokgoswami-ag**
Added 4.17 heavy build multi
``` --------------------------------------------------------------------------------------------------
4.17 heavy build multi
1 Job link: https://prow.ci.openshift.org//view/gs/test-platform-results/logs/periodic-ci-openshift-multiarch-master-nightly-4.17-ocp-heavy-build-ovn-remote-libvirt-multi-p-p/1830525960819052544
Nightly info- Failed to fetch nightly image
Lease Quota-
No crash observed
Build Passed
```
updated the cluster status error for 4.16 above builds.

### CODE REVIEW

File: p_periodic.json [hunk 1]
```diff
@@ -22,7 +21,7 @@
     "4.16 to 4.17 upgrade":"periodic-ci-openshift-multiarch-master-nightly-4.17-upgrade-from-nightly-4.16-ocp-ovn-remote-libvirt-multi-p-p",
     "4.17 libvirt": "periodic-ci-openshift-multiarch-master-nightly-4.17-ocp-e2e-ovn-remote-libvirt-ppc64le",
```
Comment (Shilpa-Gokul) [inline p_periodic.json:22]: Add entry for this also- https://prow.ci.openshift.org/job-history/gs/origin-ci-test/logs/periodic-ci-openshift-multiarch-master-nightly-4.17-ocp-e2e-ovn-remote-libvirt-multi-p-p

File: monitor.py [hunk 2]
```diff
@@ -240,13 +241,19 @@ def cluster_deploy_status(spy_link):
...
     else:
+        pattern=r"(\d+\.\d+)"
+        match=re.search(pattern,spy_link[8:])
+        version=float(match.group(1))
         job_log_url = PROW_VIEW_URL + spy_link[8:] + '/artifacts/' + job_type + '/ipi-install-' + job_platform +'-install/finished.json'
         if "sno" in spy_link:
             job_log_url = PROW_VIEW_URL + spy_link[8:] + '/artifacts/' + job_type + '/upi-install-powervs-sno/finished.json'
         
         # temporary fix for getting cluster deployment status of jobs where clusters are deployed by upi
         if "4.16-ocp-e2e-ovn-remote-libvirt-ppc64le" in spy_link:
```
Comment (Shilpa-Gokul) [inline monitor.py:252]: Do we need this line if we are checking for version > 4.16? We can remove this check right ?

Comment (alokgoswami-ag) [reply monitor.py:252]: No, we can modify match pattern for upgrade jobs . 
4.15-4.16 upgrade . here version will be 4.15 acc to our scripts. but it is 4.16

File: p_periodic.json (hunk 1 above)
Comment (alokgoswami-ag) [reply p_periodic.json:22]: ok

**Review by Shilpa-Gokul — APPROVED** [review]
/lgtm

**Pull request merged**